src/ec_git_vsn.erl

```diff
@@ -99,16 +99,10 @@ parse_tags(Pattern) ->
             {Tag, Vsn1}
     end.
 
--ifdef(unicode_str).
 len(Str) -> string:length(Str).
 trim(Str, right, Chars) -> string:trim(Str, trailing, Chars);
 trim(Str, left, Chars) -> string:trim(Str, leading, Chars).
```

Comment:
same question here. Can we or should we remove these intermediary calls? Any blockers?

Reply:
No blocker.